docs/reference/ois/next/processing.md

```diff
@@ -245,4 +333,24 @@ endpoints: [
 ]
 ```
 
+## Example #3
+
+One of the possible use cases for post-processing would be to use override the
```

Comment:
Will this also work for RRP?

Reply:
No, the timestamp is only used for signed data responses [here](https://github.com/api3dao/airnode/blob/02d15254dfc431b4c0aa06b36f64953c550849fd/packages/airnode-node/src/api/index.ts#L277)

Reply:
Any particular reason

Reply:
To be honest I am not sure what you mean. The RRPv0 doesn't have use timestamp in its [fulfill function](https://github.com/api3dao/airnode/blob/processing-v2/packages/airnode-protocol/contracts/rrp/AirnodeRrpV0.sol#L226).

Reply:
At this point I forgot how RRP works :skull: